src/Resources/config/doctrine/ProductBundle.orm.yml

```diff
@@ -0,0 +1,28 @@
+SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundle:
+    type: entity
+    table: solutiondrive_product_bundle
```

Comment:
shouldn't the table be solutiondrive_product_bundle**s**?

Reply:
![image](https://user-images.githubusercontent.com/16880583/35770273-466fabc0-0918-11e8-9a13-6ec501e11585.png)

the sylius standard tables are singular too

Reply:
ah, ok. than we can roll with it...;-)